### PR TITLE
Create a7a1729c-3626-4bfe-85da-8e4a5e865c6f.yml

### PR DESCRIPTION
**Pull request by VivekKamisetty**
id: a7a1729c-3626-4bfe-85da-8e4a5e865c6f 
metadata:
  version: 1
  authors: 
    - Vivek Kamisetty (Mr@UnKnOwN)
  tags: []
name: Fetch Disk Space 
description: |
  Displays the free space on the machine.
tactic: discovery
technique:
  id: T1082
  name: System Information Discovery
platforms:
  darwin:
    sh:
      command: df -h
  linux:
    sh:
      command: df -h
  windows:
    pwsh:
      command: wmic diskdrive get size 
    psh:
       command: wmicdiskdrive get size

### CODE REVIEW

File: discovery/a7a1729c-3626-4bfe-85da-8e4a5e865c6f.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  name: System Information Discovery
+platforms:
+  darwin:
+    sh:
+      command: df -h
+  linux:
+    sh:
+      command: df -h
+  windows:
+    pwsh:
```
Comment (privateducky) [inline discovery/a7a1729c-3626-4bfe-85da-8e4a5e865c6f.yml:22]: "pwsh" refers to PowerShell Core.. which this should work on fine i'd imagine.. but you should also add "psh" for regular powershell, which may have been your original goal. so it'd look like this:
```
windows:
    pwsh:
        command: ...
    psh:
        command: ...
```

**Comment by VivekKamisetty** [comment]
@privateducky Done ✅.

**Comment by VivekKamisetty** [comment]
So can it be merged @privateducky ?

**Comment by VivekKamisetty** [comment]
Done @privateducky

**Review by privateducky — APPROVED** [review]
Good! Once @khyberspache approves this can be merged.

**Review by khyberspache — APPROVED** [review]

**Pull request merged**